api/checks/jwt.go

```diff
@@ -58,6 +58,7 @@ func getJWTClient(ctx context.Context, appID, installation int64) (*http.Client,
 		return nil, fmt.Errorf("cannot fetch installation token: %w", err)
 	}
 	if c := resp.StatusCode; c < 200 || c > 299 {
+		// nolint:exhaustruct // TODO: Fix exhaustruct lint error.
```

Comment:
This one still needs to be added to the .yaml file and have this comment removed.

Reply:
Hi @jcscottiii, this is the last issue remaining!

I believe this struct should not be excluded. I looked through RFC 6749 and I believe ErrorCode is a required field, instead I've added a comment to investigate which error code should be returned here.